server.h

```diff
@@ -277,6 +364,15 @@ class CTFOServer final {
       card_entry.ctfo_count = card.ctfo_count;
       card_entry.tfu_count = card.tfu_count;
       card_entry.skip_count = card.skip_count;
+      card_entry.favorited = false;
+      try {
+        const EntryWrapper<Favorite> fav = favorites.Get(uid, card.cid);
```

Comment:
Yes! Removed!

You made me curious to look into, and after 1.5 hours of looking through stack traces like  http://pastebin.com/dTrF20LE, and adding https://github.com/c5t/Current/pull/231, it turned out to be a missing Cereal's registerer. Thanks!